server/middleware/fetchLicenceMiddleware.ts

```diff
@@ -1,16 +1,43 @@
 import { RequestHandler } from 'express'
+import getUrlAccessByStatus from '../utils/urlAccessByStatus'
 import LicenceService from '../services/licenceService'
+import logger from '../../logger'
 
 export default function fetchLicence(licenceService: LicenceService): RequestHandler {
   return async (req, res, next) => {
     if (req.params.licenceId) {
       const { user } = res.locals
       const licence = await licenceService.getLicence(req.params.licenceId, user)
 
+      // Does this prison user have a caseload which allows access this licence?
+      if (licence && user?.nomisStaffId) {
```

Comment:
Can user be undefined? That would suggest they are not logged in?

Reply:
Could be a probation user, in which case would be undefined.